packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -1904,7 +1917,7 @@ class WebpackCLI implements IWebpackCLI {
       path: new WeakMap(),
     };
 
-    if (options.config && options.config.length > 0) {
+    if (options.config && options.config.length > 0 && disableInterpret) {
```

Comment:
I think we don't need `disableInterpret` here

Reply:
@alexander-akait 

That's right. 

Removed: https://github.com/webpack/webpack-cli/pull/3329/commits/d4673f1f45a4581cd9da6ab1bd4c187748a35e90